package.json

```diff
@@ -8,11 +8,13 @@
   },
   "scripts": {
     "algolia": "node ./algolia.js",
+    "pre-build": "npm-run-all clean ignore gulp rollup maybe-sync-external",
```

Comment:
I think `maybe-sync-external` needs to be moved before `gulp` in this list. Right now, the `lhci` GitHub action is failing because `gulp` is run and there's no `external/data/.hash` present: https://github.com/GoogleChrome/developer.chrome.com/runs/6210054958?check_suite_focus=true#step:6:34